src/modules/universal-discovery/components/content-creator/content.creator.component.js

```diff
@@ -24,7 +24,15 @@ export default class ContentCreatorComponent extends Component {
 
     handlePublish() {
         this._refIframe.contentWindow.onbeforeunload = () => {};
-        this._refIframe.contentWindow.document.body.querySelector('#ezrepoforms_content_edit_publish').click();
+        this.submitForm();
+    }
+
+    submitForm() {
+        const submit = this._refIframe.contentWindow.document.body.querySelector('[data-action="submit"]');
```

Comment:
I think we should use more meaningful name and maybe move this to correct `Type` class?

Reply:
Not sure what you mean by that @ViniTou , but I would just change the `submit` value to `publish`.

Reply:
@Nattfarinn 
to add it here instead of twig view.
`\EzSystems\RepositoryForms\Form\Type\Content\ContentEditType`